src/attached_probe.cpp

```diff
@@ -976,6 +986,14 @@ void AttachedProbe::attach_kprobe(bool safe_mode)
     funcname = modname + ":" + funcname;
   }
 
+  // If the user requested to ignore warnings on non-existing probes and the
```

Comment:
Super nit to fix our comments - turn multi line into:
```
/*
 */
```

Reply:
We use both variants for multi-line comments (I think what I have is "C++ style") so I'd keep it as it is.

Reply:
From what I understand you can use '//' for multi-line comments in C++ but I don't think that's the preferred style. I'm trying to move us to use '/**/' for multiline.